packages/core/useInputCaretPosition/index.browser.test.ts

```diff
@@ -0,0 +1,54 @@
+import { page, userEvent } from '@vitest/browser/context'
+import { mount } from '@vue/test-utils'
+import { describe, expect, it } from 'vitest'
+import { defineComponent, nextTick } from 'vue'
+import { useInputCaretPosition } from '.'
+
+const simpleInput = defineComponent({
+  template: `<input data-testId="input" value="vueuse is cool!">`,
+})
+
+describe('useInputCaretPosition', () => {
+  it('should be defined', () => {
+    expect(useInputCaretPosition).toBeDefined()
+  })
+
+  it('should return the initial position', async () => {
+    const wrapper = mount(simpleInput)
+    const { position } = useInputCaretPosition(wrapper.element)
+    expect(position.value).toMatchInlineSnapshot(`15`)
+  })
+
+  it('should update the position selection change', async ({ onTestFailed, onTestFinished }) => {
+    const screen = page.render(simpleInput)
+    onTestFailed(() => screen.unmount())
+    onTestFinished(() => screen.unmount())
```

Comment:
tests are sometimes flaky if i dont do this. not sure why. 

I think if a test fails the page is not unmounted leaving the next tests with some unexpected initial state.